HLTrigger/Configuration/python/customizeHLTTrackingForPhaseI2017.py

```diff
@@ -312,6 +312,19 @@ def _copy(old, new, skip=[]):
 
     return process
 
+# modify the HLT configuration to run the Phase I tracking in the particle flow sequence
+def customizeHLTForPFTrackingPhaseI2017tuningIter0(process):
+    process.hltPixelTracks.OrderedHitsFactoryPSet.maxChi2.pt1    = cms.double( 0.8 ) # lowpt
+    process.hltPixelTracks.OrderedHitsFactoryPSet.maxChi2.pt2    = cms.double( 2   ) # highpt
+    process.hltPixelTracks.OrderedHitsFactoryPSet.maxChi2.value1 = cms.double( 150 ) # chi2lowpt
+    process.hltPixelTracks.OrderedHitsFactoryPSet.maxChi2.value2 = cms.double(  50 ) # chi2highpt
+    process.hltPixelTracks.OrderedHitsFactoryPSet.CAThetaCut  = cms.double( 0.002 )
+    process.hltPixelTracks.OrderedHitsFactoryPSet.CAPhiCut    = cms.double( 0.2   )
+    process.hltPixelTracks.OrderedHitsFactoryPSet.CAHardPtCut = cms.double( 0     )
```

Comment:
The modifications above are for the old seeding framework, so I'd expect tests to fail.

With the new seeding framework you should to modify `process.hltPixelTracksHitQuadruplets` instead of `process.hltPixelTracks.OrderedHitsFactoryPSet` (parameters themselves are the same so a simple replace should be enough).